samples/secret-rotation/index.js

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 Google LLC
+//
```

Comment:
Hi there! Thanks for adding samples! Would you mind following sample guidelines internally? You can search for `Node.js Sample Policies` internally, or feel free to ping me so I can share the link with you :)

Reply:
Thanks I'll take a look!